service/sg-windows/sgservice.go

```diff
@@ -0,0 +1,123 @@
+// +build windows
+package main
+
+import (
+	"log"
+	"os/exec"
+
+	"github.com/kardianos/service"
+	"os"
+)
+
+var logger service.Logger
+
+type program struct{
+	ExePath 	string
+	ConfigPath  string
```

Comment:
The types should line up vertically, looks like this wasn't run through `gofmt`yet?